product_docs/docs/odbc_connector/13.1.0.2/01_whats_new.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: "What’s New"
```

Comment:
Let's use the new release note model for the connector release notes if we have time to do so.

Reply:
@drothery-edb, I have added the release notes with the new format. It can be reviewed once it is deployed. After review, I will remove the What's New section.